userspace/libscap/engine/bpf/scap_bpf.c

```diff
@@ -1531,6 +1531,7 @@ int32_t scap_bpf_load(
 		int ret;
 		struct scap_device *dev;
 
+		/* We suppose that CPU 0 is always online, so we only check for j > 0 */
```

Comment:
(I am aware this is out of scope for this PR :))

Reply:
uhm that's a good point, not sure why we do this check, maybe @FedeDP remember this (?)

Reply:
the only reason that comes to my mind is that CPU 0 doesn't have the online file under `/sys/devices/system/cpu/cpu0/` I think that this is the initial assumption we have done... not sure if there are systems in which this CPU can go down :/

Reply:
and if i remember well all the hotplug stuff in the driver is sent on CPU 0 assuming that this will be always online, so probably to be CPU agnostic we should have logic in the driver to send the HOTPLUG event on a random online CPU